golink.go

```diff
@@ -169,18 +170,78 @@ func Run() error {
 	if err := srv.Start(); err != nil {
 		return err
 	}
-	localClient, _ = srv.LocalClient()
 
-	l80, err := srv.Listen("tcp", ":80")
+	// create tsNet server and wait for it to be ready & connected.
```

Comment:
this comment does not belong here.

Reply:
(addressed in subsequent commit) thanks!